### PR TITLE
Prevent the SequelizeDatabaseError

### PR DESCRIPTION
**Pull request by thekepaag**
I just added a note to avoid the SequelizeDatabaseError on bot startup !

### CODE REVIEW

**Comment by eartharoid** [comment]
This addition is not helpful because it doesn't explain *how* to fix it, and it's also unclear that this is the solution to a specific problem that many users won't have.

As the bot automatically creates the database tables, I think it would be better to make the bot do this on a table level, which can be done by adding these options:

```js
charset: 'utf8',
dialectOptions: {
	 collate: 'utf8_general_ci'
},
```

to both options objects where Sequelize is initiated at https://github.com/discord-tickets/bot/blob/main/src/database/index.js#L37-L51.

I don't know if that only works for MySQL or if `utf8_general_ci` is also valid for SQLite and PostgreSQL.
You can open another PR to add this to the bot (after testing with PostgreSQL) if you want or update this PR to show the complete solution and make it clear what problem the solution is for.

File: docs/using-mysql.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 A MySQL database is recommended for improved performance and allowing you to have ticket transcripts/archives. SQLite is too slow to support storing messages.
 
+!!! danger "WARNING"
```
Comment (eartharoid) [inline docs/using-mysql.md:5]:
```suggestion
!!! failure "Troubleshooting" 
    If you encounter a "SequelizeDatabaseError" when you start the bot, ...(solution)...
```

**Review by eartharoid — CHANGES_REQUESTED** [review]

File: docs/using-mysql.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 A MySQL database is recommended for improved performance and allowing you to have ticket transcripts/archives. SQLite is too slow to support storing messages.
 
+!!! danger "WARNING" 
+
+	For be sure to don't have Sequelize error on bot startup use the 'utf8_unicode_ci' collation for your database.
```
Comment (RooRay) [inline docs/using-mysql.md:7]: Please check your English is proper too that'd be appreciated though we can help with this if you need us to 😄

**Review by RooRay — CHANGES_REQUESTED** [review]
Just some spell checking also Earth's requested additions that are needed I think

**Pull request closed without merging**